spec/rubocop/cop/rspec/factory_bot/association_style_spec.rb

```diff
@@ -0,0 +1,133 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::FactoryBot::AssociationStyle do
+  def inspected_source_filename
+    'spec/factories.rb'
+  end
+
+  let(:cop_config) do
+    { 'EnforcedStyle' => enforced_style }
+  end
+
+  context 'when EnforcedStyle is :implicit' do
+    let(:enforced_style) do
+      :implicit
+    end
```

Comment:
It seems to me that it would be better to have a cop that specifies the style of the block for `let`. Would it be possible to specify which style should be used in which case?

```ruby
let(:cop_config) do
  { 'EnforcedStyle' => enforced_style }
end

let(:enforced_style) { :implicit }
```

Reply:
"Use your best judgement and your common sense" is the rule we're using now.